parler/utils/fields.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models.constants import LOOKUP_SEP
+from django.db import models
+import django.db.models.fields
+
+
+class NotRelationField(Exception):
+    pass
+
+
+def get_model_from_relation(field):
+    # type: (django.db.models.fields.Field) -> models.Model
+    if hasattr(field, 'get_path_info'):
+        return field.get_path_info()[-1].to_opts.model
+    else:
+        raise NotRelationField
+
+
+def get_extra_related_translalation_paths(model, path):
```

Comment:
typoo in the function name!

Reply:
Also needs test coverage for this function (TODO)